docs/data/material/getting-started/templates/dashboard/components/HighlightedCard.tsx

```diff
@@ -29,13 +29,18 @@ const Card = styled(MuiCard)(({ theme }) => ({
 
 export default function HighlightedCard() {
   return (
-    <Card variant="highlighted" sx={{ height: '100%' }}>
+    <Card
+      variant="highlighted"
+      sx={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+      }}
+    >
       <CardContent>
         <InsightsIcon color="primary" />
-        <Typography
-          sx={{ fontSize: 14, flexGrow: 1, fontWeight: 'medium' }}
```

Comment:
Ideally we should try to use the default typography styles, or tweak the token directly in the theme, this way we can be sure that everything is consistent